app/javascript/packs/public.js

```diff
@@ -133,9 +133,12 @@ function main() {
 
   delegate(document, '#account_display_name', 'input', ({ target }) => {
     const name = document.querySelector('.card .display-name strong');
-
     if (name) {
-      name.innerHTML = emojify(target.value);
+      if (target.value) {
+        name.textContent = emojify(target.value);
```

Comment:
textContent = emojify doesn't work, since emojify returns img tags for custom emoji

Reply:
:+1:

Reply:
Okay, I think there may be a wider bug, but even on `master` I cannot see the emojis in this field. I checked out https://writing.exchange/ and https://chaos.social/ and it doesn't work there, either.